jetty-jmx/src/test/java/org/eclipse/jetty/jmx/ObjectMBeanTest.java

```diff
@@ -123,7 +123,7 @@ public void testDerivedAttributes() throws Exception
 
         assertEquals("com.acme.Derived", derivedInfo.getClassName(), "name does not match");
         assertEquals("Test the mbean stuff", derivedInfo.getDescription(), "description does not match");
-        assertEquals(6, derivedInfo.getAttributes().length, "attribute count does not match");
+        assertEquals(5, derivedInfo.getAttributes().length, "attribute count does not match");
```

Comment:
I don't understand why this value changed here? Doesn't seem anything to do with shutdown?

Reply:
the `stopTimeout` used to be an attribute on every AbstractLifeCycle. It no longer is, hence one less attribute.